src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java

```diff
@@ -509,6 +509,37 @@ public SBApiResponse getCBPlanListForUser(String userOrgId, String authUserToken
         return response;
     }
 
+    public SBApiResponse requestCbplanConten(SunbirdApiRequest request, String userOrgId, String token){
```

Comment:
Follow the existing implementation style.
check userId is not empty - if empty return 400 error
have a validation method to check competency and providerList is not null from the request - if either is null return 400 error

Reply:
This is implemented please verify.